mat/matrix_test.go

```diff
@@ -693,3 +693,73 @@ func TestDoer(t *testing.T) {
 		}
 	}
 }
+
+func TestGetIndexedView(t *testing.T) {
+	for testnum, test := range []struct {
+		a             Matrix
+		rows          []int
+		cols          []int
+		accessRows    []int
```

Comment:
It would be nicer if this were a `[]struct { r, c int; want float64 }`.

Reply:
Maybe I misunderstood what you meant here, but I did some changes which I think addresses this.

Reply:
Yes, that change is essentially what I meant. It would be a little cleaner if you define a throwaway type to avoid having to repeat the type definition for each case.